docs/stories/03-inputs/Combobox.stories.tsx

```diff
@@ -137,6 +163,170 @@ export const Loading: Story = {
   },
 };
 
+export const WithStickyFooter: Story = {
+  render: () => {
+    const [isLoading, setIsLoading] = React.useState(false);
+    const handleLoadMore = () => setIsLoading(true);
+
+    React.useEffect(() => {
+      if (isLoading) {
+        const timeout = setTimeout(() => setIsLoading(false), 2000);
+        return () => clearTimeout(timeout);
+      }
+    }, [isLoading]);
+
+    const handleClickButton = () => {
+      // Handle button click to open modal or perform an action
+      console.log('Button clicked');
+    };
+
+    // Create a sticky footer with a button that triggers the modal
+    const createRelationTrigger = (
+      <Button variant="ghost" startIcon={<Plus />} onClick={handleClickButton} fullWidth justifyContent={'start'}>
+        Create a relation
```

Comment:
I wouldn't mention a create action here to avoid confusion with the actual creatable state of the combobox. Can you just remove the Button and just put a <Typography>Footer</Typography> here please.